### PR TITLE
Add Example Number 1

### PR DESCRIPTION
**Pull request by Maninda**
## Purpose
Resolves https://github.com/ballerina-platform/ballerina-standard-library/issues/2079

## Checklist
- [x] Linked to an issue
- [x] Updated the changelog
- [ ] Added tests

### CODE REVIEW

File: examples/gmail-smtp-client/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+## Testing
+
+We can run the above developed code, in our local environment. Navigate to
+[`examples/gmail-smtp-client/client`](./client) directory and execute the following command.
+```shell
+$ bal run
+```
+
+The successful execution of the service should show us the following output.
```
Comment (praneesha) [inline examples/gmail-smtp-client/README.md:46]:
```suggestion
The successful execution of the service should show the output below.
```

File: examples/gmail-smtp-client/README.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+```shell
+Compiling source
+	smtp/gmail_sender:1.0.0
+
+Running executable
+```
+
+Now, check the inbox of the email receiver. Sent email should be available after some time.
```
Comment (praneesha) [inline examples/gmail-smtp-client/README.md:54]:
```suggestion
Now, check the inbox of the email receiver. The sent email should be available after some time.
```

File: examples/gmail-smtp-client/client/gmail_sender.bal [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+import ballerina/email;
+
+configurable string senderAddress = ?;
+configurable string receiverAddress = ?;
+configurable string senderPassword = ?;
+
+public function main() returns error? {
+
+    // Creates an SMTP client to Gmail. The default port number `465` is
```
Comment (praneesha) [inline examples/gmail-smtp-client/client/gmail_sender.bal:25]:
```suggestion
    // Creates an SMTP client for Gmail. The default port number `465` is
```

File: examples/gmail-smtp-client/client/gmail_sender.bal [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    // used over SSL with these configurations.
+    email:SmtpClient smtpClient = check new ("smtp.gmail.com",
+        senderAddress, senderPassword);
+
+    // Defines the email that is required to be sent. `'from` address is
```
Comment (praneesha) [inline examples/gmail-smtp-client/client/gmail_sender.bal:30]:
```suggestion
    // Defines the email that is required to be sent. The `'from` address is
```

File: changelog.md [hunk 5]
```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Added
+- [[#2079] Add example number 1](https://github.com/ballerina-platform/ballerina-standard-library/issues/2079)
```
Comment (shafreenAnfar) [inline changelog.md:9]: I think we don't need to updated the change log as this is not a functional change.

**Review by praneesha — APPROVED** [review]

File: changelog.md (hunk 5 above)
Comment (Maninda) [reply changelog.md]: Removed.

**Review by praneesha — APPROVED** [review]

**Pull request merged**